### PR TITLE
fix: remove finalization

### PR DESCRIPTION
**Pull request by ronag**

### CODE REVIEW

**Comment by ronag** [comment]
@mcollina semver?

File: README.md [hunk 1]
```diff
@@ -203,13 +203,7 @@ not support or does not fully implement.
...
 
 The [Fetch Standard](https://fetch.spec.whatwg.org) allows users to skip consuming the response body by relying on
-[garbage collection](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management#garbage_collection) to release connection resources. Undici does the same. However,
-garbage collection in Node is less aggressive and deterministic (due to the lack
-of clear idle periods that browser have through the rendering refresh rate)
-which means that leaving the release of connection resources to the garbage collector
-can lead to excessive connection usage, reduced performance (due to less connection re-use),
-and even stalls or deadlocks when running out of connections. Therefore, it is highly
-recommended to always either consume or cancel the response body.
+[garbage collection](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management#garbage_collection) to release connection resources. Undici does not do the same. Therefore, it is important to always either consume or cancel the response body.
```
Comment (mcollina) [inline README.md:206]: We might want to explain why we cannot do this.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
I'd go with minor.

**Pull request merged**